src/lib/urlHelper.js

```diff
@@ -12,6 +12,22 @@ export function getNewEntryUrl(collectionName, direct) {
   return getUrl(`/collections/${ collectionName }/entries/new`, direct);
 }
 
+// Unreserved chars from RFC3987.
```

Comment:
Can you be a little bit more verbose in this comment, maybe even add a reference link? You did a lot of research, just want to capture some of it here.

Reply:
Sure, because I sure don't want to have to do it over again :).